src/Numerics/DGMethods/DGModel.jl

```diff
@@ -15,6 +15,15 @@ struct DGModel{BL, G, NFND, NFD, GNF, AS, DS, HDS, D, DD, MD}
     modeldata::MD
 end
 
+DGModel(driver_config; kwargs...) = DGModel(
+    driver_config.bl,
+    driver_config.grid,
+    driver_config.numerical_flux_first_order,
+    driver_config.numerical_flux_second_order,
+    driver_config.numerical_flux_gradient;
+    kwargs...,
+)
+
```

Comment:
If we move this to belong in the Driver module instead of the DGMethods module, wouldn't that solve anything? I thought one of the advantages of Julia and multiple dispatch was that you can extend a module from the outside.

Reply:
You're quite right, I see no issue with that.

Reply:
Ah, good point! I'll reopen and move it